src/services/query.ts

```diff
@@ -66,6 +66,9 @@ export function renderLogQLFieldFilters(filters: AdHocVariableFilter[]) {
 }
 
 function renderFilter(filter: AdHocVariableFilter) {
+  if (filter.value === '""') {
```

Comment:
Would it break everything if the value is literally an empty string? I dislike seeing the "" in the field.

Reply:
Probably, i'll take a look tho

Reply:
So yeah, empty string means no filter in the variable, and is evaluated falsey in a bunch of places. This is the correct value for the query, I think we'd need more control over adhoc variables to get this to work

Reply:
I imagined that was a possibility. Thanks for checking!